projects/admin/src/app/circulation/patron/pending/pending.component.ts

```diff
@@ -46,4 +50,18 @@ export class PendingComponent implements OnInit {
       }
     });
   }
+
+  /**
+   * Remove the canceled request on the loans list.
+   *
+   * @param loanId, the canceled loan id
+   */
+  cancelRequest(loanId: any): void {
+    // Remove loan in list
+    const index = this.loans.findIndex((element: any) => element.id == loanId);
+    this.loans.splice(index, 1);
+    // Update count on tab
+    this._circulationService.circulationInformations.statistics['pending'] =
+      this._circulationService.circulationInformations.statistics['pending'] -1;
```

Comment:
shorter : `this._circulationService.circulationInformations.statistics['pending'] -= 1` ?